static/js/biomedical/protein/protein_protein_interaction_graph.tsx

```diff
@@ -83,8 +95,10 @@ export class ProteinProteinInteractionGraph extends React.Component<
 
   componentDidUpdate(prevProps: Props, prevState: State): void {
     if (
+      !this.state.showTableView &&
```

Comment:
there seems to be a lot of logic to determine whether or not to draw a graph, maybe make a helper function like `shouldDrawGraph()` that returns a boolean to replace the stuff in the `if`

Reply:
ah sounds good will do!